requirements/test

```diff
@@ -0,0 +1,4 @@
+-r base
+pytest
+pytest-cov
+pytest-runner
```

Comment:
the PR to remove pytest-runner has been merged

Reply:
ty, will fix....I thought I resolved all conflicts but missed one!